sdk/src/impl/whirlpool-client-impl.ts

```diff
@@ -149,8 +152,8 @@ export class WhirlpoolClientImpl implements WhirlpoolClient {
               pos.whirlpool,
               this.ctx.program.programId
             ).publicKey;
-            tickArrayAddresses.add(lowerTickArrayPda);
-            tickArrayAddresses.add(upperTickArrayPda);
+            tickArrayAddresses.add(lowerTickArrayPda.toBase58());
```

Comment:
this used to insert duplicate addresses, since two `PublicKey` objects with the same addresses were considered to be different